service/controller/resource/statuscondition/create.go

```diff
@@ -142,23 +142,84 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 }
 
 func (r *Resource) computeCreateClusterStatusConditions(ctx context.Context, cl apiv1alpha2.Cluster, cr infrastructurev1alpha2.CommonClusterObject, nodes []corev1.Node, controlPlanes []infrastructurev1alpha2.G8sControlPlane, machineDeployments []apiv1alpha2.MachineDeployment) error {
```

Comment:
hmm maybe we can also rename the function to computeClusterStatusConditions and writeClusterConditions

Reply:
don't forget to update the CHANGELOG.md 🙂

Reply:
Right, thank you for the heads up. :)
And yeah, that naming would fit better. I'll use that.